### PR TITLE
AYR-1010 - Setup Pa11y CI as a custom pre-commit hook

### PR DESCRIPTION
**Pull request by ltrasca**
## Changes in this PR
- Added a configuration file for pa11y-ci inside configs/ containing the URLs that should be tested
- Added a bash script that calls pa11y-ci via npx
- Included an entry inside .pre-commit-config.yaml that calls the bash script mentioned above as a custom pre-commit hook

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-1010

### Before
N/A

### After
N/A

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: custom_hooks/pa11y_ci_precommit.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set +e
+
+export $(grep -v '^#' .env.e2e_tests | xargs)
+
+# Install pa11y-ci if not installed
+npm list | grep pa11y-ci || npm install --save-dev pa11y-ci --no-shrinkwrap
```
Comment (colinbowen) [inline custom_hooks/pa11y_ci_precommit.sh]:
```suggestion
OUTPUT=$(npx pa11y-ci --config configs/pa11y_ci_precommit.js)
echo "${OUTPUT}"
```

Comment (colinbowen) [reply custom_hooks/pa11y_ci_precommit.sh]: Might be able to remove the cmd from package.json if you use npx

**Review by anthonyhashemi — APPROVED** [review]
Nice one 👌

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -57,6 +57,15 @@ repos:
         language: system
         pass_filenames: false
 
+-   repo: local
+    hooks:
```
Comment (anthonyhashemi) [inline .pre-commit-config.yaml:61]: Not for now, but like we discussed if we decide we want to have it just as a pre-push hook if we find it takes too long on each commit, we can specify stages https://pre-commit.com/#config-stages

**Pull request merged**